### ISSUE TITLE
Error occurred when run serverless deploy

### ISSUE THREAD

**Issue by sdruipeng** [serverless/serverless#1853]
I am using serverless.1.0.0-beta.1.1 and following the document.

The error report is: cannot create the S3 bucket.

Yet, there is really no setup for the S3 in serverless.yml

It is a surprise that I got stuck from the very beginning. 

This error shows every time I try

**Comment by pmuens** [comment]
Hey @sdruipeng so you're deploying a simple Serverless service and are not using a S3 event?

Have you setup your AWS user so that he can create the necessary resources on your behalf as described here: https://github.com/serverless/serverless/blob/master/docs/guide/installation.md?

**Comment by flomotlik** [comment]
@sdruipeng the bucket it needs to create is for the deployment itself. We're deploying Serverless application artefacts through an S3 bucket, so your user accounts needs to rights to create it.

As this is more a support issue and less a Serverless core problem you can also get more help in our Forum at http://forum.serverless.com

**Comment by sdruipeng** [comment]
**@ pmuens 
@ flomotlik

Thank you for you information.**
I checked my IAM user again, making sure that it all conforms to the instructions described at provider-account-setup.md. *But I got the error as follows again.*

By the way, I have developed a website based on S3 and react/redux, I am planning to use the serverless to do the Login for my website. Yet, I got stuck from the very beginning at serverless.


$ serverless deploy
Serverless: Zipping service...

  Serverless Error ---------------------------------------

     The specified bucket does not exist

  Get Support --------------------------------------------
     Docs:          v1.docs.serverless.com
     Bugs:          github.com/serverless/serverless/issues
